community/tlp/post-install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+cat <<EOF
+
+NOTE: TLP requires a flock version with the w flag.
+      If your version does not support this you
+      could switch to another version, like
+      util-linux.
+
+EOF
```

Comment:
tlp needs patching to fix this issue.

Reply:
Here's the one place where `-w` is used: https://github.com/linrunner/TLP/blob/e83b6ac6b136d09fe35a9f7277f0bb7717cc82b6/tlp-func-base.in#L240

Reply:
I added a patch to solve this issue